.circleci/workflows.yml

```diff
@@ -453,8 +443,10 @@ commands:
           command: yarn versions
       - unless:
           condition:
-            # stop-only does not correctly match on windows: https://github.com/bahmutov/stop-only/issues/78
-            equal: [ *windows-executor, << parameters.executor >> ]
+            or:
+              # stop-only does not correctly match on windows: https://github.com/bahmutov/stop-only/issues/78
```

Comment:
I think we should just remove this check to be honest but out of scope here

Reply:
Reverted this change since it appears it was addressed on develop.